tidy3d/components/heat/simulation.py

```diff
@@ -116,24 +111,66 @@ def check_unsupported_geometries(cls, val):
                 raise SetupError(
                     f"'HeatSimulation' does not currently support structures with dimensions of zero size ('structures[{ind}]')."
                 )
+        return val
+
+    @staticmethod
+    def _check_cross_solids(objs, values):
+        """Given model dictionary ``values``, check whether objects in list ``objs`` cross
+        a ``SolidSpec`` medium.
+        """
 
-            if isinstance(structure.geometry, GeometryGroup):
-                geometries = structure.geometry.geometries
+        # list of structures including background as a Box()
+        structure_bg = Structure(
+            geometry=Box(
```

Comment:
just wondering if we have any sort of check in case these are not in `values`. If so whether we should error gracefully. (maybe the validator / methods that call this already handle it?)